docs/sources/writing-guide/front-matter/index.md

```diff
@@ -143,17 +146,16 @@ For more detail about HTML redirects, refer to [HTML redirections](https://devel
 
 > **Note:** The redirect relies on first party JavaScript support which is common but not necessarily universal.
 
-## Guidelines
+### Guidelines
 
-### Unversioned projects:
+The correct way to use aliases depends on whether the project is versioned or not.
 
-Include an `aliases` entry that refers to the initial published website directory.
+**Unversioned projects**
+: Include an `aliases` entry that refers to the initial published website directory.
 Adding an `aliases` entry makes it safer to move content around as the redirect from old to new page location is already in place.
 Hugo doesn't create a redirect `.html` file when the directory is already populated with content.
+: > **Note:** The published directory is dependent on which `content` subdirectory documentation is synced to in the website repository.
```

Comment:
Sorry, should have mentioned in my previous comments. It looks like more lines start with a `:`

Reply:
That's how to make description list `<dd>` tags in Goldmark. On the page I built locally, it rendered like this:

https://user-images.githubusercontent.com/47385188/211001327-3dc63bb1-0651-4e0a-96ee-d1c66e4c9ad6.png

Reply:
I guess GFM doesn't like it -- https://github.com/grafana/writers-toolkit/pull/166/files?short_path=2e4f9ad#diff-2e4f9adf86d9ba3dcbd3a49683af50fb530e1a473f9338b6958321106e6f41f5

Reply:
Yeah, IMO local/staging builds are the way to go. There always will be little discrepancies. And github markdown itself breaks sometimes,  e.g. with custom anchors--- `# heading {#custom-slug}`--- no longer working.  

The hugo extensions on top of common mark are here: https://github.com/yuin/goldmark/tree/master/extension

We did add description list syntax to the toolkit, but there's might be a few others worth documenting from there.